paseos/paseos.py

```diff
@@ -47,7 +47,6 @@ def __init__(self, local_actor: BaseActor):
 
     def advance_time(self, time_to_advance: float):
         """Advances the simulation by a specified amount of time
```

Comment:
same as in spacecraft actor, these white spaces are part of [the docstring format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)